core/src/main/resources/error/error-classes.json

```diff
@@ -1062,6 +1062,11 @@
     ],
     "sqlState" : "42604"
   },
+  "INVALID_URL_STRING" : {
```

Comment:
Could you make it shorter `INVALID_URL`, please. The word `_STRING` doesn't bring any additional info, IMHO.